RSDKv5/RSDK/Input/SDL2/SDL2InputDevice.cpp

```diff
@@ -208,7 +208,7 @@ RSDK::SKU::InputDeviceSDL *RSDK::SKU::InitSDL2InputDevice(uint32 id, uint8 contr
         controllerType = DEVICE_XBOX;
     else if (strstr(name, "PS4") || strstr(name, "PS5"))
         controllerType = DEVICE_PS4;
-    else if (strstr(name, "Nintendo") || strstr(name, "Switch")) {
+    else if (strstr(name, "Nintendo") || strstr(name, "Switch") || strstr(name, "Wii U")) {
```

Comment:
Left-over from Wii U port.

Reply:
Oh no, it benefits PC users as well: SDL2's controller database contains entries for the Wii U Pro Controller, so this will make those be recognised as Switch controllers, which I think is a better fit than recognising them as Xbox controllers.

Reply:
Honestly, fair enough. Works for me.